source/code/include/playfab/PlayFabApiSettings.h

```diff
@@ -11,20 +11,15 @@ namespace PlayFab
     class PlayFabApiSettings
```

Comment:
>PlayFabApiSettings [](start = 10, length = 18)

Do you anticipate this class to be copy or move constructed / assigned?

Reply:
Copied: Very much no. We might go so far as to completely disable that option...
Moved: Probably not.
Shared: Yes, we will very definitely need to have a single shared obj, referenced by multiple instanceAPIs.

---
In reply to: [343284614](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343284614) [](ancestors = 343284614)